src/utils.ts

```diff
@@ -2,7 +2,7 @@ import { ConnectorInput } from "grindery-nexus-common-utils/dist/connector";
 import { convert } from "./web3/evm/unitConverter";
 
 // eslint-disable-next-line @typescript-eslint/no-explicit-any
-async function sanitizeObject(parameters: Record<string, unknown>, input: ConnectorInput<any>) {
+async function sanitizeObject(parameters: Record<string, unknown>, input: ConnectorInput<string | any>) {
```

Comment:
For me here the goal is not exactly this one.

1. As mentioned in [this comment](https://github.com/grindery-io/grindery-nexus-connector-web3/issues/72#issuecomment-1610989698), @SAPikachu modified the `grindery-nexus-common-utils` repo to enforce a better typing by modifying some of the type definition like `ConnectorInput` (you can have a look at the provided commit in the comment). As a result, the version of `grindery-nexus-common-utils` was bumped to 2.0.17. So first we need an update of the `grindery-nexus-common-utils` package version to have access to this new type definitions. You can update versioning in this PR.
2. As a result, the goal was to enforce better typing and avoid `any` or `unknow` types that can be unsafe. For example, here, `ConnectorInput<T>` can be set where `T` should be a type describing what is in `fields` as you can see [here](https://github.com/grindery-io/grindery-nexus-common-utils/blob/68c0bbaeedb56c0fed0baa3ae09dff5066b69495/src/connector/types.ts#L7).